engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -48,28 +48,15 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workf
 
   override def receive = {
     case ValidateWorkflow =>
-      validateWorkflow(sender())
-    // NOTE: self shuts down when the parent PerRequest shuts down
-  }
-
-  private def validateWorkflow(sentBy: ActorRef): Unit = {
-    logger.info(s"$tag for $sentBy")
-    // TODO: IMO, the responsibility for returning a status code should be a part of the client. This class should only return a validation result,
-    // and the callee should convert that to a status code or anything that it wants
-    validateAll() map {
-      case _: ValidationSuccess =>
-        logger.info(s"$tag success $sentBy")
-        sentBy ! RequestComplete(
-          StatusCodes.OK,
-          APIResponse.success("Validation succeeded."))
-
-      case ValidationFailure(reason) =>
-        val messageOrBlank = Option(reason.getMessage).mkString
-        logger.info(s"$tag error $sentBy: $messageOrBlank")
-        sentBy ! RequestComplete(
-          StatusCodes.BadRequest,
-          APIResponse.fail(reason))
-    }
+      val requester = sender()
+      logger.info(s"$tag for $requester")
```

Comment:
I agree with that, but something like that was there already before, and I did't want to remove it. Now that you said I'll remove/replace it :smile: